### PR TITLE
[Travis] Create after_deploy Honeycomb script

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] DX

## Description
This will add a marker to our Honeycomb board whenever the app is deployed to Heroku. It provides additional context used for debugging and analytics.

## Related Tickets & Documents
[Honeycomb's documentation](https://docs.honeycomb.io/api/markers/#create-a-marker)

Resolves https://github.com/orgs/forem/projects/6#card-52224759

## QA Instructions, Screenshots, Recordings
This is a curl command. I've tested it thoroughly locall.y If it doesn't work, it wouldn't cause any problem and I'll address it immediately.

here's what the feature looks like. It's the dotted line.
image)

## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -110,6 +112,7 @@ jobs:
       if: type != pull_request
       script: skip
       after_script: skip
+      after_deploy: skip
```
Comment (maestromac) [inline .travis.yml:115]: Skip after_deploy for BenHalpern Community deploy.

**Review by mstruve — APPROVED** [review]
In the future, we should figure out how this could work for all Forems but I am game for at least doing this for now for DEV. 

Given this is not used by Heroku and we are having slug issues, should we add this file to slugignore?

**Review by rhymes — APPROVED** [review]
+1 on adding it to slug ignore though it might merit a separate PR as I believe a bunch of stuff in `scripts/` can go in the slug ignore as well

**Review by andygeorge — APPROVED** [review]
Seems reasonable so :+1:, although @forem/systems doesn't manage Heroku/Travis so I don't have a super strong opinion on this (until, as Molly referenced, we're using Honeycomb for other Forems).

**Pull request merged**